safe/common/test/test_version.py

```diff
@@ -18,9 +18,11 @@
 __copyright__ += 'Disaster Reduction'
 
 
+import os
 import unittest
 import sys
 from safe.common.version import get_version, current_git_hash
+from safe.definitions.versions import inasafe_version, inasafe_release_status
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->